src/css/object/project/contents.scss

```
@@ -0,0 +1,48 @@
+@use '../../object/utility/utility.scss';
+
+.p-contents {
+  padding-top: 106px;
+  padding-bottom: 106px;
```

[nits]
`padding`や`margin`はワンライナーで書ける場合は、ワンライナーで書くことが多いので、こちらの書き方にも慣れておくと良いと思います！

```suggestion
  padding: 106px 0;
```

---

ワンライナーで、上記修正いたしました！
09cfd8f214aec06867f4d0b525f94db209ef0c53